geomstats/geometry/hypersphere.py

```diff
@@ -828,7 +887,11 @@ def squared_dist(self, point_a, point_b, **kwargs):
     def parallel_transport(
         self, tangent_vec, base_point, direction=None, end_point=None
     ):
-        r"""Compute the parallel transport of a tangent vector.
```

Comment:
When escapes are used within the docstrings, e.g. within math expressions, we need to add "r" to allow proper rendering. Notice that linting also checks it.